Libplanet.Extensions.ActionEvaluatorCommonComponents/AccountStateDelta.cs

```diff
@@ -26,6 +26,8 @@ public class AccountStateDelta : IAccountStateDelta, IValidatorSupportStateDelta
             g => g.Key,
             g => (IImmutableSet<Currency>)g.Select(kv => kv.Key.Item2).ToImmutableHashSet()
         );
+
+    public IImmutableDictionary<Address, IImmutableSet<Currency>> TotalUpdatedFungibleAssets { get; }
```

Comment:
It needs to be implemented too, but I'll open a new pull request to implement it. (Because it is related to our team's goal) (The rc-v200030 can ignore the PR).